snipsnapdev/snipsnap
# Feat/rename file

**Describe what changes this pull request brings**

Covering #92 we add a dropdown to the file with `Delete` and `Rename` options.

**Describe any additional changes**

Rename modal is generalized and now supports both file and folder.


**Steps to test**
* create a file in Snipsnap admin UI
* try to rename this file, check result persists after saving
* try to delete this file
* regression testing that folders management is not broken

**Screenshots**
<img width="452" alt="Screenshot 2021-07-23 at 02 40 54" src="https://user-images.githubusercontent.com/1070568/126714257-14c7aa43-9115-448c-8593-6104bc10b3bf.png">

<img width="783" alt="Screenshot 2021-07-23 at 02 49 02" src="https://user-images.githubusercontent.com/1070568/126714252-48a7996c-ab43-4cee-adc1-d15e99408de1.png">

**Checklist**



- [x] I've checked for typos
- [x] I've made sure overall structure consistency is preserved
- [x] I've made sure overall headings hierarchy is preserved
- [x] I've reread proposed changes twice

**TODO**



**References**
Issue #92
